xx.md

```diff
@@ -0,0 +1,56 @@
+NIP-XX
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines a new event kind `31337` representing an audio track. These events are _parameterized replaceable_
+and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+A kind `31337` event's `content` should be summary or description of the audio content.
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in NIP 92. Multiple `imeta` tags MAY be included to provide for different use cases
+  (e.g. streaming vs download).
+- `title` is the title of the audio track
+- `subject` is the title of the audio track (deprecated, but required for compatibility)
+
+The following tags are optional:
+
+- `i` is an external GUID.
+- `c` is the name of a category.
+- `creator` is the pubkey or plain text name of the track creator. This tag MAY have a mark denoting the creator's role.
+- `website` is an external url to a website related to the track.
+- `duration` is the duration of the audio track, in seconds.
+- `published_at` is a timestamp representing the track's original publish date.
+
+Any `creator` tags containing a pubkey MUST have an accompanying `p` tag so that the track creator is indexable.
+
+```json
+{
+  "id": <event_id>,
+  "pubkey": <author_pubkey>,
+  "created_at": <created_at>,
+  "kind": 31337,
+  "content": "Chill beats",
+  "tags": [
+    ["d", "<UUID>"],
+    ["c", "Chill Beats"],
+    ["creator", "<pubkey>"],
+    ["creator", "<plain text>", "feat"],
+    ["title", "Platinum Robots on the Moon"],
+    ["subject", "Platinum Robots on the Moon"],
+    ["published_at", "<published_at>"],
+    ["imeta", "url https://example.com/my-track.m3u8 x <hash> ..."],
+    ["p", "<pubkey>", "wss:/example.com", "jim"]
+  ]
```

Comment:
Per NIP-94, I think it would be useful to include the `m` tag here to indicate the data type. This would make the types more explicit for clients and help prevent content playback from breaking. Per that spec: "The [MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) format must be used, and they should be lowercase."

```suggestion
    ["p", "<pubkey>", "wss:/example.com", "jim"]
    ["m", "<MIME type>"]
  ]
```

The addition could also allow this kind to be used for a variety of media. Don't see why this should be limited to only audio.

Reply:
`imeta` includes `m`, and I think it's necessary to structure it that way because you might have multiple urls for a single track with different mime types.

Reply:
Ah I missed that reference, thanks. That looks great. Maybe adding a link to NIP-92 in the tag descriptions would be helpful?